Solution

LaskoA opened the pull request:
Please check if everything is OK so far. If yes, I will continue with Extra section. Thank you ;)

Abnormaltype commented on app/main.py:
```diff
@@ -1 +1,26 @@
+    7: 0.0,
+    8: 0.0,
+    9: 0.0,
+    10: 0.0}
```
It is bad style to write dictionary like this manually.

Abnormaltype commented on app/main.py:
```diff
@@ -1 +1,26 @@
+    for case in range(0, 10000):
+        for num, flip in enumerate(range(1, 11)):
+            flip = random.choice(option)
+            if flip == "heads":
```
and where condition when `flip =="tails"`?

LaskoA commented:
Hi :) 

1) I have changed the way of adding elements to dictionary, please check it now;
2) Do we really need to count number of "tails"? It`s not mentioned in the task at all. Or did I miss something?
3) Added "draw_gaussian_distribution_graph" function - please check it as well.

Thank you.

IvanRamyk reviewed (changes requested):
I tried to execute `flip_coin` function and got an error
Could you call your functions from the module and make sure they work correctly?

LaskoA commented:
Problem was fixed (line 14, dictionary[num] instead of num). Kindly ask you to check again. Thank you :)
